provider/bigquery.go

```diff
@@ -870,16 +876,20 @@ func (store *bqOfflineStore) RegisterResourceFromSourceTable(id ResourceID, sche
 }
 
 func (store *bqOfflineStore) RegisterPrimaryFromSourceTable(id ResourceID, tableLocation pl.Location) (PrimaryTable, error) {
+	store.logger.Infow("Registering primary from source table", "id", id.Name)
+
 	if err := id.check(Primary); err != nil {
 		return nil, err
 	}
+
 	if exists, err := store.tableExists(id); err != nil {
 		return nil, err
```

Comment:
I know this isn't your code, but could you add error logs to the error returns, and any other logs